modules/ROOT/pages/6.5-release-notes.adoc

```diff
@@ -71,6 +71,12 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
 
 {productname} 6.5 also includes the following bug fixes:
 
+=== Quickbar toolbars was shown for noneditable contents in a noneditable root.
```

Comment:
What do you think about using non-editable. I see that docs seems to prefer using `contenteditable="false"` elements over non-editable elements but I kind of like the non-editable wording better since the `contenteditable="false"` might be an internal implementation detail. We have ways to make editor elements non-editable without the need for the integrator to specifically specifying the attribute internally while editing we still add the attribute but when they save the attribute is removed again. The second thing is that I think it's a bit harder to read.

```suggestion
=== Quickbar toolbars was shown for non-editable contents in a non-editable root.
```

or

```suggestion
=== Quickbar toolbars was shown for non-editable contents in a `contenteditable="false` root.
```

Reply:
@spocke I agree, but I at the same time I know we are trying to use `contenteditable="false"` over `noneditable` to be more uniform across docs.

@brian-forte What are your thoughts on the above?